### DISCUSSION TITLE
Nodal Bottom Depth / Vertical Integration

### DISCUSSION THREAD

**Discussion by alxthor** [FESOM/fesom2#405] [Q&A] [answered]
I had a look at the core2 mesh files in order to perform vertical integration/averaging of output and also looking at the mesh bathymetry.
What is the difference between the values listed in ```aux3d.out``` (according to the [documentation](https://fesom2.readthedocs.io/en/latest/meshes/meshes.html) ```H_v``` is the depth at each vertex) and the values in the ```fesom.mesh.diag.nc``` (```zbar_n_bottom``` is described as nodal bottom depth)?
The values in the ```fesom.mesh.diag.nc``` seem to always match a FESOM level, but it is not always the closest level to the one in ```aux3d.out```.
For integration/averaging as I understand, it is fine using the levels as defined in the top part of ```aux3d.out``` since the actual zstar layer thicknesses (e.g. in the ```hnode.nc``` restart file) differ only slightly from the ones in ```aux3d.out```.

**Comment by koldunovn** [comment, marked as answer]
The levels in `aux3d.out` are given as an input to the model, that then generates depth according to the set of model levels provided. Long story short one always have to use `zbar_n_bottom`. 

Yes, you can in most cases just ignore small differences in layer thickness.
